src/pages/Accounts/Modals/CreateAccount.tsx

```diff
@@ -47,6 +47,13 @@ export const CreateAccountModal: FC<TAccountModalProps> = ({ isVisible, onFinish
     setStage(stage - 1);
   }, [stage]);
 
+  useEffect(() => {
+    if (!isVisible) {
```

Comment:
if I understand problem correctly - then you want to move this into "onClose" instead